test/logging.test.js

```diff
@@ -26,12 +26,12 @@ const logEmitterEmitSpy = jest.spyOn(logEmitter, 'emit')
 
 const exampleErrorLog = [
   {
-    ts: new Date('2018-01-01T01:01:01+01:00'),
+    ts: new Date('2018-01-01T01:01:43+01:00'),
```

Comment:
Added randomness to the seconds to distinguish formatted time's seconds from minutes